tracee-ebpf/tracee/tracee.go

```diff
@@ -124,6 +124,7 @@ type OutputConfig struct {
 	StackAddresses bool
 	DetectSyscall  bool
 	ExecEnv        bool
+	MonotonicTS    bool
```

Comment:
Since this is a user facing config, I suggest to call this (and cli flag) "Relative time" or something similar, since monotonic does not describe the essence of this flag.